spark/dl/src/test/scala/com/intel/analytics/bigdl/torch/BCECriterionSpec.scala

```diff
@@ -59,13 +55,16 @@ class BCECriterionSpec extends TorchSpec{
 
   "A BCECriterion with weights" should "generate correct output and grad" in {
     torchCheck()
-    val weights = Tensor[Double](3).rand()
+    val weights = Tensor[Double](2).rand()
```

Comment:
weights should be the same as input's second dimension.
First dimension is batch size.